### ISSUE TITLE
Can you provide the training weights for the BDCN network you are using?

### ISSUE THREAD

**Issue by 2236107323** [FangHanNUS/MM22_PIMoG#2]

**Comment by FangHanNUS** [comment]
In this paper, BDCN is fixed and only used for generating the mask of the training images. We did not re-train a new BDCN. For generating the mask, we use the default setting of BDCN.

**Comment by 2236107323** [comment]
Thank you for your reply, but I have used the official bdcn provided by BDCN_ pretrained_ on_ Bsds500.pth, it was found that the edges extracted from the image are very thick and cloudy. Do I need to perform subsequent maximum suppression operations to achieve the sample effect provided in your training set? This issue has been bothering me for a long time. If possible, I would like to receive your code for creating the dataset? Thank you.




改变一切
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "FangHanNUS/PIMoG-An-Effective-Screen-shooting-Noise-Layer-Simulation-for-Deep-Learning-Based-Watermarking-Netw"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月25日(星期二) 下午4:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [FangHanNUS/PIMoG-An-Effective-Screen-shooting-Noise-Layer-Simulation-for-Deep-Learning-Based-Watermarking-Netw] Can you provide the training weights for the BDCN network you are using? (Issue #2)





 
In this paper, BDCN is fixed and only used for generating the mask of the training images. We did not re-train a new BDCN. For generating the mask, we use the default setting of BDCN.
